arches/app/search/components/ids.py

```diff
@@ -0,0 +1,29 @@
+from arches.app.search.elasticsearch_dsl_builder import Bool, Ids
+from arches.app.search.components.base import BaseSearchFilter
+from arches.app.utils.betterJSONSerializer import JSONDeserializer
+
+details = {
+    "searchcomponentid": "f1856bfb-c3c4-4d67-8f23-0aa3eef3a160",
+    "name": "Ids Filter",
+    "icon": "",
+    "modulename": "ids.py",
+    "classname": "IdsFilter",
```

Comment:
Could you change the class name to 'ResourceIdsFilter'. Otherwise there isn't any place but the test that indicates the type of id.

Reply:
Or - add a doc string to the class to clarify